tests/Doctrine/Tests/Common/Collections/BaseCollectionTest.php

```diff
@@ -55,6 +69,16 @@ public function testMap() : void
         self::assertEquals([2, 4], $res->toArray());
     }
 
+    public function testReduce() : void
+    {
+        $this->collection->add(1);
+        $this->collection->add(2);
```

Comment:
Can you expand this with more elements please? It should make the test more expressive.

Reply:
Sure, done.